loader/src/main.py

```diff
@@ -152,7 +169,15 @@ def process_event(msg):
             loader.load(
                 resolved_fhir_instance, resource_type=resolved_fhir_instance["resourceType"],
             )
-            producer.produce_event(topic=PRODUCED_TOPIC, record=resolved_fhir_instance)
+
+            producer.produce_event(
+                topic=PRODUCED_TOPIC_PREFIX+batch_id,
+                record={
+                    "fhir_object": resolved_fhir_instance,
```

Comment:
Is the fhir_object field used in any consumer for this topic?

Reply:
Nop.. I can remove it unless we see a use case.